packages/babel-helpers/src/helpers.js

```diff
@@ -413,6 +416,29 @@ helpers.interopRequireWildcard = template(`
   })
 `);
 
+helpers.specRequireInterop = template(`
+  (function (obj) {
+    if (obj && obj.__esModule) {
+      return obj;
+    } else {
+      var newObj = Object.create(null, {
```

Comment:
This helper is only called when using the `spec: true` option, which can only generate exports using `Object.defineProperty`. I couldn't find any platform with a working `Object.defineProperty` but not a working `Object.create` with a quick look at compat-table...